hack/build-rpms.sh

```diff
@@ -28,6 +28,11 @@ tar czf "${rpm_tmp_dir}/SOURCES/${OS_RPM_NAME}-test.tar.gz" \
     .
 cp -r "${ci_data}/." "${rpm_tmp_dir}/SOURCES"
 
+# for file in /etc/yum.repos.d/*; do sed "s#vault.centos.org/centos/7#vault.centos.org/centos/7.8.2003#g" $file -i; done
+echo $releasever
```

Comment:
From the error in the CI job seems the script is run with set -u
```suggestion
echo ${releasever-""}
```